src/esp/assets/ResourceManager.cpp

```diff
@@ -1168,7 +1168,8 @@ scene::SceneNode* ResourceManager::createRenderAssetInstanceIMesh(
 
     // Instance mesh does NOT have normal texture, so do not bother to
     // query if the mesh data contain tangent or bitangent.
-    gfx::Drawable::Flags meshAttributeFlags{};
+    gfx::Drawable::Flags meshAttributeFlags{
+        gfx::Drawable::Flag::HasVertexColor};
```

Comment:
Yes, this is very necessary. Thanks.

Reply:
@ldcWV have you tested this change?

What I'm imagining: 1. find a suitable test model that uses this codepath (I can provide one, let me know). 2. On your local system, put a breakpoint or print statement here to sanity-check that you're testing this codepath (createRenderAssetInstanceIMesh). 3. Run locally and verify the model is rendered correctly.

Reply:
Yep, Yili gave me a .ply vertex colored model to test this. Without the flag everything is white, but adding the flag makes the model colors viewable.
<img width="801" alt="Screen Shot 2021-06-03 at 1 43 50 PM" src="https://user-images.githubusercontent.com/4172272/121066893-435eea80-c77f-11eb-8dcf-f4d0cafc8fde.png">
<img width="795" alt="Screen Shot 2021-06-03 at 2 32 12 PM" src="https://user-images.githubusercontent.com/4172272/121066899-4528ae00-c77f-11eb-95c2-8042f5109c0d.png">

Reply:
Is it supposed to be all purple? Can we find another test model or otherwise verify this is correct? @bigbike thoughts?

Reply:
Oh, I see. The issue is marked as resolved. That is why I did not see it. 

Yes, it is supposed to be all purple.
Yes, we can find another one to test.